src/v/raft/consensus.cc

```diff
@@ -2011,4 +2045,37 @@ void consensus::suppress_heartbeats(
     }
 }
 
+voter_priority consensus::next_target_priority() {
+    return voter_priority(
+      std::max<int32_t>(_target_priority * target_priority_update_factor, 1));
+}
+
+/**
+ * We use simple policy where we calculate priority based on the position of the
+ * node in configuration broker vector. We shuffle brokers in raft configuration
+ * so it should give us fairly even distribution of leaders across the nodes.
+ */
+voter_priority consensus::get_node_priority(model::node_id id) const {
+    auto latest_cfg = _configuration_manager.get_latest();
+    auto& brokers = latest_cfg.brokers();
+
+    auto it = std::find_if(
+      brokers.cbegin(), brokers.cend(), [id](const model::broker& b) {
+          return b.id() == id;
+      });
+
+    if (it == brokers.cend()) {
+        return voter_priority(1);
```

Comment:
What is priority 1? what does this mean really. Is this highest? or lowest, why not `voter_priority::max()` or smth like that

what is `0` we need a comment here.

Reply:
done